api/src/main/resources/workbench.yaml

```diff
@@ -967,6 +995,59 @@ definitions:
         format: int64
         description: Milliseconds since the UNIX epoch.
 
+  MaterializeCohortRequest:
+    type: object
+    properties:
+      cohortName:
+        description: >
+         The name of a cohort that is to be evaluated. Either this or cohortSpec should be specified
+        type: string
+      cohortSpec:
+        description: >
+          JSON representation of a cohort to be evaluated (using the same format used for saved
+          cohorts). Either this or cohortName should be specified
+        type: string
+      statusFilter:
+        description: >
+          An array of status values; participants with these statuses will be included.
+          Defaults to [NOT_REVIEWED, INCLUDED, NEEDS_FURTHER_REVIEW] -- everything but EXCLUDED.
```

Comment:
I'm not sure how this maps to the cohort definition since the criteria are opaque JSON. Should there be further hints here?

Reply:
It doesn't map to the cohort definition -- these are cohort statuses, populated from our review UI.

Reply:
For a bit more info, see:

https://docs.google.com/document/d/1JrzQgxq9vB42sHKwMDI8CKzAADNY7r7WUfVQX3mc8Nw/edit#heading=h.da49ri852lfy